sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -595,21 +598,31 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     const messageBody: any[] = [];
     for (let i = 0; i < messages.length; i++) {
       const item = messages[i];
-      if (!item.messageId) item.messageId = generate_uuid();
-      item.scheduledEnqueueTimeUtc = scheduledEnqueueTimeUtc;
 
       try {
-        const amqpMessage = toRheaMessage(item, defaultDataTransformer);
-
-        const entry: any = {
-          message: RheaMessageUtil.encode(amqpMessage),
-          "message-id": item.messageId
+        const rheaMessage = toRheaMessage(item, defaultDataTransformer);
+        updateMessageId(rheaMessage, rheaMessage.message_id || generate_uuid());
+        updateScheduledTime(rheaMessage, scheduledEnqueueTimeUtc);
+
+        const entry: {
```

Comment:
Always love seeing `any` go away.

Reply:
Yeah, it was kind of unnecessary but at the same time it bothered me so I figured, why not? :)